src/TLabel.cpp

```diff
@@ -131,12 +164,36 @@ void TLabel::mouseMoveEvent(QMouseEvent* event)
 {
     if (forwardEventToMapper(event)) {
         return;
+    } else if (mpHost) {
+        TEvent tmpMoveParams = mMoveParams;
+        for (auto button : mMouseButtons.keys()) {
+            if (button & event->buttons()) {
+                tmpMoveParams.mArgumentList.append(mMouseButtons.value(button));
+                tmpMoveParams.mArgumentTypeList.append(ARGUMENT_TYPE_STRING);
```

Comment:
As above about other buttons.

Reply:
Nope nope nope please no, separate function instead, if you need that information you can get it. Otherwise we have bloated documentation that's difficult to use for a newbie too just because it has _so much stuff_.

Reply:
As above - events are transitory so subsequently trying to recall the situation that applied after an event requires a much more substantive effort to do in code, if it _is_ possible...

That is not to say that there is not potentially separate requirement for a _polling_ mouse interface that answers the questions: where is the mouse now and what buttons are currently down?  That however, is a different kettle of worms...